requirements-base.txt

```diff
@@ -3,3 +3,4 @@ oauthlib>=1.0.3
 requests-oauthlib>=0.6.1
 six>=1.10.0
 PyJWT>=1.4.0
+cryptography>=2.1.1
```

Comment:
Where's this used?

Reply:
Please check https://github.com/python-social-auth/social-core/pull/155#discussion_r157049337